src/System.Private.CoreLib/shared/System/Diagnostics/Tracing/EventSource.cs

```diff
@@ -3126,14 +3136,20 @@ private unsafe bool SendManifest(byte[] rawManifest)
                     // For large manifests we want to not overflow any receiver's buffer. Most manifests will fit within
                     // 5 chunks, so only the largest manifests will hit the pause.
                     if ((envelope.ChunkNumber % 5) == 0)
+                    {
+#if PROJECTN
+                        RuntimeThread.Sleep(15);
```

Comment:
Similar for other PROJECTN ifdefs. There should be very little difference between CoreCLR and CoreRT (ideally none).

Also, PROJECTN define is turned on in the internal ProjectN TFS build only. We may need something else for it.